fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/HyperGraphComparator.java

```diff
@@ -73,7 +73,7 @@ public class HyperGraphComparator {
     private final Map<Edge, List<? extends Expression>> pullUpQueryExprWithEdge = new HashMap<>();
     private final Map<Edge, List<? extends Expression>> pullUpViewExprWithEdge = new HashMap<>();
     private final LogicalCompatibilityContext logicalCompatibilityContext;
-    private final Map<JoinEdge, Pair<JoinType, Set<Slot>>> inferredViewEdgeWithCond = new HashMap<>();
+    private final Map<JoinEdge, Pair<JoinType, Set<Set<Slot>>>> inferredViewEdgeWithCond = new HashMap<>();
```

Comment:
could u add some comment later to explain what mean about all variable's key and value?

Reply:
use Pair rather than Set